app/models/team.rb

```diff
@@ -53,6 +53,10 @@ def by_season_phase
         Team.visible.where season: Season.current
       end
     end
+
+    def active_teams
+      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
```

Comment:
I can't use the :by_season_scope, because of the visibility thingy in there. Hence the newly added scope here for the team query, and a season query in the season model.